qiling/extensions/trace.py

```diff
@@ -2,14 +2,16 @@
 
 # More info, please refer to https://github.com/qilingframework/qiling/pull/765
 
+from __future__ import annotations
```

Comment:
I don't think there is a circular dependency between `trace` and `Qiling`, so I don't think this is actually needed here.